index.html

```diff
@@ -14,320 +14,320 @@
+          <h1 class="greetings">I'm Rassel</h1>
+        </div>
+        <div class="top-page-profession">
+          <h3 class="activity">I'm a Software Developer</h2>
+        </div>
+        <div class="header3">
+          <p class="top-page-description">
+            I can help you build a product, feature or website. Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to contact me.
+          </p>
         </div>
-        <h1 class="greetings">I'm Rassel</h1>
-      </div>
-      <div class="support-text1">
-        <h3 class="activity">I'm a Software Developer</h2>
       </div>
-      <div class="header3">
-        <p class="description1">
-          I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
-      </p>
+      <div class="top-page-social-media">
+        <ul id="top-page-social-media-list">
+          <li>
+            <a href="#">
+              <img src="./images/11a_git-icon.png" class="icon git" alt="git-icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/12a_linkedin-icon.png" class="icon linkedin" att="linjedin icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/13a_victory-icon.png" class="icon victory" alt="victory icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/14a_twitter-icon.png" class="icon twitter" alt="twitter icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/15a_m-icon.png" class="icon gmail" alt="gmail icon">
+            </a>
+          </li>
+        </ul>
       </div>
-    </div>
-    <div class="social-media">
-      <ul id="horizontal-list">
-        <li>
-          <a href="#">
-            <img src="./images/git-icon.png" class="icon git" alt="git-icon">
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="./images/linkedin-icon.png" class="icon linkedin" att="linjedin icon">
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="./images/victory-icon.png" class="icon victory" alt="victory icon">
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="./images/twitter-icon.png" class="icon twitter" alt="twitter icon">
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="./images/m-icon.png" class="icon gmail" alt="gmail icon">
-          </a>
-        </li>
-      </ul>
-    </div>
-  </section>
+    </section>
+  <!--TOP PAGE ENDS HERE-->
+
+  <!--MY RECENT WORKS STARTS HERE-->
 
-  <section class="mid-section">
+  <section class="my-recent-work">
```

Comment:
[OPTIONAL] Your `works` section still looks a bit different from the [design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=56ISm4d7DE3nsZgj-0)😉 kindly use the right color as given in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=56ISm4d7DE3nsZgj-0)😉😉 Don't forget, we are required to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=56ISm4d7DE3nsZgj-0) in this milestone👍 kindly take note of the difference in the image be low😉

yours
<img width="957" alt="design" src="https://user-images.githubusercontent.com/64643995/221373037-f6ad8f29-acee-4a62-8a8e-547a61a873ba.png">


design
<img width="551" alt="yours" src="https://user-images.githubusercontent.com/64643995/221373013-4d18ba7e-c902-4437-a0cd-ffc6c6d70163.png">

kindly maintain the `white` color😉